docs/tethys_sdk/extensions/templates_and_static_files.rst

```diff
@@ -4,20 +4,22 @@ Templates and Static Files
 
 **Last Updated:** February 22, 2018
 
-Templates and static files in extensions can be used in other apps.  The advantage to using templates and static files from extensions in your apps is that when you update the template or static file in the extension, all the apps that use them will automatically be updated. Just as with apps, store the templates in the ``templates`` directory and store the static files (css, js, images, etc.) in the ``public`` directory. Then reference the template or static file in your app's controllers and templates using the namespaced path.
+Templates and static files in extensions can be used in other apps.  The advantage to using templates and static files from extensions in your apps is that when you update the template or static file in the extension, all the apps that use them will automatically be updated. Just as with apps, store the templates in the ``templates`` directory and store the static files (css, js, images, etc.) in the ``public`` directory. When using an extension template in your app's controllers, import the extension and use it to render the templates.
 
 For example, to use an extension template in one of your app's controllers:
 
 ::
 
+    from tethysext.my_first_extension import Extension as MyFirstExtension
+
     def my_controller(request):
         """
         A controller in my app, not the extension.
         """
         ...
-        return render(request, 'my_first_extension/a_template.html', context)
+        return MyFirstExtension.render(request, 'a_template.html', context)
```

Comment:
@swainn In the scaffold I named the `TethysExtensionBase` subclass as `Extension`, but I'm open to calling it `Ext`. 

In this example I'm explicity importing the extension as `MyFirstExtension`. Since it's being imported as a "3rd-party" package, and since apps could potentially use multiple extensions, it seemed like good practice to import extensions as a specific name:

```python
from tethysext.my_first_app.ext import Extension as MyFirstExtension
````

Reply:
Ah, I see. That makes sense.

Reply:
Re: Extension vs. Ext. let's just keep it as Extension as you have it.